odata-protocol/10 Context URL.md

```diff
@@ -103,6 +103,7 @@ the type of the returned entity collection.
 Context URL template:
 
     {context-url}#{entity-set}/$entity
+    {context-url}#{single-navprop}
```

Comment:
In line 25 above we define

> - `{entity-set}` is the name of an entity set or path to a containment navigation property

Please add a definition for `{single-navprop}` and reduce the definition of `{entity-set}` to collection-valued nav props.